lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -71,6 +72,32 @@ def self.create_release(repository, version, release_notes, assets, prerelease)
           github_client().upload_asset(release[:url], file_path, content_type: 'application/octet-stream')
         end
       end
+
+      # Downloads a file from the given GitHub release
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] relase The title of the GitHub release
```

Comment:
```suggestion
      # @param [String] release The title of the GitHub release
```

Technically, the `release` parameter is used in a place of the URL where the commit-ish is expected, not the title of the GitHub release. You can have a GitHub release titled `"The best update ever"` pointing to a tag named `"v1.2"`, and what you pass when you construct the githubusercontent URL is the tag name `v1.2` not the title. So this doc might be misleading.

In fact, that part of the URL expects any "commitish" (as git calls them), so it could be a tag name like you designed it to here, but it would also work with a commit sha1, or a branch name. (And maybe one day we might want to download a file from a specific branch rather than tag, so that should be valid)

```suggestion
      # @param [String] commit The git tag name, branch name, or commit sha1 to download
```

Reply:
Right. But does `release` make more sense here?. We are going to use this for downloading from releases, so doesn't passing a release title to a parameter which is called `commit` sound confusing?

Reply:
We're not passing a release **title** though. We're passing a tag name 😉  So keeping this named as `release` is _more_ confusing to me than calling it `commit`, because if you were to pass the release title to that parameter this would not work for GH releases that happens to not have the same name as the tag they're attached to ([example](https://github.com/realm/SwiftLint/releases/tag/0.43.1)).

I agree that the name `commit` might not be the nicest one, but I feel that we should still use a more appropriate name – e.g. maybe name this parameter `tag:` instead?

Reply:
Yeah, I understand your point, but it's just something we don't do. Anyway, exactly because we don't do it, I think it's not worth it to spend more time discussing this :-) I've switched it to tag in `6223829`.